llvm/test/CodeGen/AMDGPU/llvm.amdgcn.fdot2.bf16.bf16.ll

```diff
@@ -72,4 +72,23 @@ entry:
   ret void
 }
 
+; FIXME: This test violates constant bus restriction.
```

Comment:
So this test will fail the verifier? Does it need to start in a separate xfailed test?

Reply:
No, it does not. This is considered a legal immediate. It is legal indeed, but only with op_sel_hi selecting low 16 bits. There is code in the SIFoldOperands to do so, but if this code is not run for any reason isImmLegal (or whatever it is) says its is legal and we just happily inline it.

Reply:
I.e. this is a bug very hard to run into.